### PR TITLE
Post html form

### PR DESCRIPTION
**Pull request by seoulection**

### CODE REVIEW

File: src/main/java/webserver/router/Router.java [hunk 1]
```diff
@@ -1,38 +1,88 @@
...
+            String indexedPath = "/todo/" + (todoItemList.size() + 1);
+            TodoPageCreator.createTodoPage(path, title);
+            String body = TodoListBuilder.buildItem(title);
+            HashMap<String, String> pageDescriptors = HtmlBuilder.createPageDescriptors(title, body);
+            addRoute(new Route("GET", indexedPath, HtmlBuilder.createHtmlString(pageDescriptors)));
+            addRoute(new Route("HEAD", indexedPath, HtmlBuilder.createHtmlString(pageDescriptors)));
+            todoList.add(new TodoItem(indexedPath, title));
+            updateRoute("/todo", TodoListBuilder.buildList(todoItemList));
+            return createPostResponse(httpRequest, SEE_OTHER, indexedPath);
+        }
```
Comment (omarnyte) [inline src/main/java/webserver/router/Router.java]: This method seems to be doing quite a bit of work. I would suggest abstracting the most nested part of this method into a private helper method.

File: src/main/java/webserver/router/Route.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        return path;
+    }
+
+    public String getBody() {
+        return body;
+    }
+
+    public void setBody(String newBody) {
+        this.body = newBody;
+    }
```
Comment (omarnyte) [inline src/main/java/webserver/router/Route.java]: I wouldn't generally think that a route contains a body. I tend to think of routes as messengers that move requests and responses back and forth. Could you explain your reasoning for giving your routes bodies?

File: src/main/java/webserver/router/Router.java (hunk 1 above)
Comment (seoulection) [reply src/main/java/webserver/router/Router.java]: This method was very problematic for me, but was refactored in the refactor PR!

File: src/main/java/webserver/router/Route.java (hunk 2 above)
Comment (seoulection) [reply src/main/java/webserver/router/Route.java]: The "body" was used to associate response data to a route. This has been refactored in the refactor PR as well!

**Pull request merged**